TLM/TLM/UI/SubTools/LaneArrows/LaneArrowTool.cs

```diff
@@ -422,40 +423,45 @@ public LaneArrowTool(TrafficManagerTool mainTool)
         /// </summary>
         void IOnscreenDisplayProvider.UpdateOnscreenDisplayPanel() {
             if (fsm_ == null) {
-                OnScreenDisplay.Clear();
+                OnscreenDisplay.Clear();
```

Comment:
`OnscreenDisplay.Clear()` appears to always be called in this method; would it be worth separating it out of this `if` statement, so:

```cs
void IOnscreenDisplayProvider.UpdateOnscreenDisplayPanel() {
    OnscreenDisplay.Clear();

    if (fsm_ == null) {
        return;
    }

    ...
}
```

The `OnscreenDisplay.Clear()` at end of mehtod could then also be removed?

Reply:
This should only be called if fsm is null (no tool mode active) or if state does not have onscreen hints. I do not want to call it for modes where the modes have hints, it hides the keybinds panel and invalidates it just to show it again in the next line.